### PR TITLE
Separate step and slab size initialization for LTS

### PR DESCRIPTION
**Pull request by wthrowe**
### Types of changes:

- [ ] Bugfix
- [ ] New feature

### Component:

- [ ] Code
- [ ] Documentation
- [ ] Build system
- [ ] Continuous integration

### Code review checklist

- [ ] The PR passes all checks, including unit tests, `clang-tidy` and `IWYU`. For
  instructions on how to perform the CI checks locally refer to the [Dev guide
  on the Travis CI](https://spectre-code.org/travis_guide.html).
- [ ] The code is documented and the documentation renders correctly. Run `make doc`
  to generate the documentation locally into `BUILD_DIR/docs/html`. Then open
  `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).

### CODE REVIEW

File: src/Evolution/DiscontinuousGalerkin/InitializeElement.hpp [hunk 1]
```diff
@@ -270,13 +270,25 @@ struct InitializeElement {
     static auto initialize(
         db::DataBox<TagsList>&& box,
         const Parallel::ConstGlobalCache<Metavariables>& cache,
-        const Time& initial_time, const TimeDelta& initial_dt) noexcept {
+        const double initial_time_value,
+        const double initial_dt_value) noexcept {
       using Vars = typename variables_tag::type;
       using DtVars = typename dt_variables_tag::type;
 
+      const bool time_runs_forward = initial_dt_value > 0;
```
Comment (nilsdeppe) [inline src/Evolution/DiscontinuousGalerkin/InitializeElement.hpp]: I guess this should be 0.0?

File: src/Time/Tags.hpp [hunk 2]
```diff
@@ -136,8 +136,9 @@ struct InitialTime {
 /// \ingroup TimeGroup
 /// \brief The initial time step taken by the time stepper. This may be
 /// overridden by an adaptive stepper
-struct DeltaT {
+struct InitialTimeStep {
```
Comment (nilsdeppe) [inline src/Time/Tags.hpp:139]: Big :+1: on this change :)

**Review by nilsdeppe — CHANGES_REQUESTED** [review]
I'm happy for you to squash the one little thing :)

**Comment by wthrowe** [comment]
I also found another `0.0` and alphabetized the list of registration functions at the bottom of EvolveBurgers.hpp.

**Review by nilsdeppe — APPROVED** [review]

**Review by kidder — APPROVED** [review]

**Comment by kidder** [comment]
travis failures are timeouts

**Pull request merged**